.claude/skills/skill-developer/TRIGGER_TYPES.md

```
@@ -0,0 +1,311 @@
+
+**Module Creation:**
+- User prompt: "create a database module"
+- Matches: `(create).*?(module)` (if module in pattern)
+- Activates: `python-dev-guidelines`
+
+### Best Practices
+
+- Capture common action verbs: `(create|add|modify|build|implement)`
+- Include domain-specific nouns: `(feature|endpoint|component|workflow)`
+- Use non-greedy matching: `.*?` instead of `.*`
+- Test patterns thoroughly with regex tester (https://regex101.com/)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)